src/utils/product-utils.ts

```diff
@@ -9,9 +9,7 @@ export function isOutOfStock(
 ): boolean {
     if (product.manageVariants) {
         const selectedVariant = getSelectedVariant(product, selectedChoices);
-        if (selectedVariant?.stock?.trackQuantity === true) {
-            return selectedVariant?.stock?.quantity === 0;
-        } else {
+        if (selectedVariant !== undefined) {
```

Comment:
wait what  ??
3 days ago @yurii-ve did the change which you removed

Reply:
Something went wrong in his reasoning.. but he said it's not going to happen again

Reply:
I mean @yurii-ve 😄